libpromises/changes_chroot.h

```diff
@@ -25,6 +25,11 @@
 #ifndef CFENGINE_CHANGES_CHROOT_H
 #define CFENGINE_CHANGES_CHROOT_H
 
+#define CHROOT_CHANGES_LIST_FILE "/changed_files"
+#define CHROOT_RENAMES_LIST_FILE "/renamed_files"
```

Comment:
@vpodzime So this means I cannot create and manage a file at `/changed_files` with cfengine, and use audit mode, right?

Could we store it in memory instead to avoid this?

Or in workdir outside of chroot?

Or at least check if that file exists and error if you try to start audit mode with a file `/changed_files` or `/renamed_files` ?

Reply:
> Or at least check if that file exists and error if you try to start audit mode with a file `/changed_files` or `/renamed_files` ?

This is the only thing I'd be willing to do. But let's not overcomplicate things for a theoretical corner case like this in a non-critical/debugging agent mode.

Reply:
I guess another thing we could do is make it really explicit / clear by calling them `cf_audit_changed_files` and `cf_audit_renamed_files`.